.gitignore

```diff
@@ -10,6 +10,7 @@
 *.seed
 *.sublime-project
 *.sublime-workspace
+*.png
```

Comment:
Do we definitely want to ignore _every_ .png file?  What if we (or anyone) wants a png image in the public dir?